installation/README.md

```diff
@@ -1,11 +1,9 @@
 # Installation
 
-Some simple guides to setting up the software on your Raspberry Pi.
+We recommend using [Raspberry Pi Imager](https://www.raspberrypi.org/software/) to install an operating system on your Raspberry Pi. As well as Raspberry Pi OS, you can use it to install other operating systems such as Ubuntu, and LibreELEC (Kodi media centre).
```

Comment:
"Ubuntu, and LibreELEC" should probably be "Ubuntu or LibreELEC" ?

Reply:
Fixed.